docs/topics/multiplatform-mobile/multiplatform-mobile-setup.md

```diff
@@ -37,12 +37,14 @@ We recommend that you install the latest stable versions for compatibility and b
    </tr>
    <tr>
         <td><a href="multiplatform-mobile-plugin-releases.md">Kotlin Multiplatform Mobile plugin</a></td>
-        <td>In Android Studio, select <strong>Settings/Preferences | Plugins</strong>, search <strong>Marketplace</strong> for <i>Kotlin Multiplatform Mobile</i>, and then install it.</td>
+        <td>In Android Studio, select <menupath>Settings/Preferences | Plugins</menupath>, search <control>Marketplace</control> for <i>Kotlin Multiplatform Mobile</i>, and then install it.</td>
```

Comment:
Does this change anything?
I see nothing wrong with the `<strong/>` tag

Reply:
I'm following the style guidelines for main tags provided in our documentation tool. I suppose the downside for using <strong> everywhere is that if designers decide to change the rendering for UI controls or menu paths, we won't get this update automatically

Reply:
There is a bold highlight for all other UI elements in the other places (like `** File** | **New project**`).
So, changing the rendering by webhelp team will lead to a conflict with markdown markup.